AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/README.md

```diff
@@ -55,82 +45,74 @@ The sample uses pretrained model provided by Intel and published as part of [Int
-> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
->
-> For more information on configuring environment variables, see *[Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html)*.
-
-#### Activate Conda
-
-1. Activate the conda environment:
-   ```
-   conda activate pytorch
-   ```
-
-2. Activate conda environment without Root access (Optional).
-
-   By default, the Intel AI Tools are installed in the `/opt/intel/oneapi` folder and require root privileges to manage them.
-
-
-   You can choose to activate Conda environment without root access. To bypass root access to manage your Conda environment, clone and activate your desired Conda environment using the following commands similar to the following.
-   ```
-   conda create --name user_pytorch --clone pytorch
-   ```
-   Then activate your conda environment with the following command:
-   ```
-   conda activate user_pytorch
-   ```
-#### Run the Script
-
-1.	Navigate to the directory with the sample.
-    ```
-    cd ~/oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted
-    ```
-2. Run the Python script.
-   ```
-   python Intel_Extension_For_PyTorch_Hello_World.py
-   ```
-   You will see the DNNL verbose trace after exporting the `DNNL_VERBOSE`:
-   ```
-   export DNNL_VERBOSE=1
-   ```
-   >**Note**: Read more information about the mkldnn log at [https://oneapi-src.github.io/oneDNN/dev_guide_verbose.html](https://oneapi-src.github.io/oneDNN/dev_guide_verbose.html).
-
-### Run the Jupyter Notebook
-
-1. Change to the sample directory.
-2. Launch Jupyter Notebook.
-   ```
-   jupyter notebook --ip=0.0.0.0 --port 8888 --allow-root
-   ```
-3. Follow the instructions to open the URL with the token in your browser.
-4. Locate and select the Notebook.
-   ```
-   ResNet50_Inference.ipynb
-   ```
-5. Change your Jupyter Notebook kernel to **PyTorch**.
-6. Run every cell in the Notebook in sequence.
-
-### Troubleshooting
-
-If you receive an error message, troubleshoot the problem using the **Diagnostics Utility for Intel® oneAPI Toolkits**. The diagnostic utility provides configuration and system checks to help find missing dependencies, permissions errors, and other issues. See the *[Diagnostics Utility for Intel® oneAPI Toolkits User Guide](https://www.intel.com/content/www/us/en/develop/documentation/diagnostic-utility-user-guide/top.html)* for more information on using the utility.
-
-
-### Example Output
+
+
+## Run the Sample
+>**Note**: Before running the sample, make sure [Environment Setup](https://github.com/ZhaoqiongZ/oneAPI-samples/tree/development/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted#environment-setup) is completed.

```

Comment:
@ZhaoqiongZ This is pointing to your fork. Can you update it and also use a relative path?